tests/Fixtures/e2e/Custom_tmp_dir/phpunit.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
```

Comment:
aren't those the default? You can easily find out with phpstorm and adding `xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.4/phpunit.xsd"
 ` e.g. https://github.com/humbug/box/blob/master/phpunit.xml.dist#L3